includes/template-tags.php

```diff
@@ -608,6 +610,126 @@ function dokan_store_category_menu( $seller_id, $title = '' ) {
+            foreach ( $terms as $term ) {
+                $current_values = isset( $_chosen_attributes[ $taxonomy ]['terms'] ) ? $_chosen_attributes[ $taxonomy ]['terms'] : [];
+                $option_is_set  = in_array( $term->slug, $current_values, true );
+                $filter_name    = 'filter_' . wc_attribute_taxonomy_slug( $taxonomy );
+                // phpcs:ignore
+                $current_filter = isset( $_GET[ $filter_name ] ) ? explode( ',', wc_clean( wp_unslash( $_GET[ $filter_name ] ) ) ) : array();
+
+                if ( ! in_array( $term->slug, $current_filter, true ) ) {
+                    $current_filter[] = $term->slug;
+                }
+
+                $link = remove_query_arg( $filter_name, $store_url );
+                // Add current filters to URL.
+                foreach ( $current_filter as $key => $value ) {
+                    // Exclude query arg for current term archive term.
+                    if ( $value === dokan_get_current_term_slug() ) {
+                        unset( $current_filter[ $key ] );
+                    }
+
+                    // Exclude self so filter can be unset on click.
+                    if ( $option_is_set && $value === $term->slug ) {
+                        unset( $current_filter[ $key ] );
+                    }
+                }
+
+                if ( ! empty( $current_filter ) ) {
+                    asort( $current_filter );
+                    $link = add_query_arg( $filter_name, implode( ',', $current_filter ), $link );
+
+                    // Add Query type Arg to URL.
+                    if ( 'or' === $query_type && ! ( 1 === count( $current_filter ) && $option_is_set ) ) {
+                        $link = add_query_arg( 'query_type_' . wc_attribute_taxonomy_slug( $taxonomy ), 'or', $link );
+                    }
+                    $link = str_replace( '%2C', ',', $link );
+                }
+
+                $checked = '';
+                if ( $option_is_set ) {
+                    $checked = 'checked';
+                }
+
+                echo '
+                <li class="parent-cat-wrap">
+                    <a href=' . esc_url( $link ) . '> <input style="pointer-events: none;" type="checkbox" ' . $checked . '> &nbsp;' . $term->name . ' <span style="float: right" class="count">(' . $term->count . ')</span> </a>
+                </li>';
+            }
+            echo '</ul>';
+        }
+        ?>
+    </div>
+        <?php
+    }
+
+endif;
+/**
+ * Return the currently viewed term slug.
+ *
+ * @return int
+ */
+function dokan_get_current_term_slug() {
+    return absint( is_tax() ? get_queried_object()->slug : 0 );
+}
+
+/**
+ * Get chosen taxonomy attributes.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @return array
+ */
+function dokan_get_chosen_taxonomy_attributes() {
+    $attributes = [];
+    // phpcs:disable WordPress.Security.NonceVerification.Recommended
+    if ( empty( $_GET ) ) {
```

Comment:
Is this check needed? If `$_GET` is empty, `foreach` loop should not run automatically.